### DISCUSSION TITLE
computer-engineering-masters-exam

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#5] [Announcements]
# https://aprd.ir/computer-engineering-masters-exam/

امیر پورمند هستم. رتبه یک کنکور ارشد شدم. میخوام به عنوان دانشجویی که خودش اول راهه یه سری چیزهایی که تجربه کردم رو باهاتون به اشتراک بذارم…

https://aprd.ir/computer-engineering-masters-exam/

**Comment by pourmand1376** [comment]
این کامنت از سایت Graphcomment - که قبلا استفاده می‌کردم - استخراج شده است. 

**نویسنده کامنت:** «سمیرا»
**متن کامنت:**
عالی بود ممنونیم بابت به اشتراک گذاشتن این اطلاعات ارزشمند

**Comment by pourmand1376** [comment]
**نویسنده کامنت:** «الهه فرج زاده»
**متن کامنت:**
سلام ،من دانشجوی سال اول مهندسی پزشکی هستم ولی خیلی دوست دارم المپیاد مهندسی کامپیوتر شرکت کنم،و میخوام از الان بخونم تا ۳،۲ سال آینده  
میخوام بدونم که آیا میتونم از رشته ی دیگه ای شرکت کنم؟ ممنونم. 

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
سلام الهه خانم
فکر نکنم بتونید المپیاد مهندسی کامپیوتر رو شرکت کنید. با این حال از مسئول استعداد درخشان دانشگاهتون هم بپرسید. لطفا اگر امکانش هست اینجا هم بگید چون سوال خیلی‌ها هست.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:** [آقای چالش - ویرگول](https://virgool.io/@mrchallenge.ir)
**متن کامنت:**
ممنون از مطالب مفید و جامع و کامل:)  
برای آزمون استخدامی من دارم ارشد میخونم یکی از بزرگترین مشکلاتم هم شبکه و امنیتت شبکه هست که میخاستم تا اونجا که بشه زبان اصلی نخونم(وقت گیره!) پیشنهاد شما چیه برای این شبکه گسترده:؟

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
سلام. خواهش می‌کنم.
قبول دارم شبکه خیلی گسترده است. باید گستره مطالعه‌ات محدود بشه که بتونی بخونیش. برای محدود کردن، اگر بتونی نمونه سوال برای آزمون استخدامی پیدا کنی، خیلی بهتر میشه فهمید. شبکه دو تا رویکرد بالا به پایین (کراس راس) و پایین به بالا (تننباوم) داره.
احتمال میدم آزمون‌های استخدامی خیلی از کنکور قدیمی‌تر باشند و هنوز رویکرد قدیم (پایین به بالا - تننباوم) رو پیش بگیرند. اگر فرض من درست باشه برای تننباوم کتاب تست زیاد هست و می‌تونی راحت جمعش کنی ولی اگر بالا به پایین باشه مرجع همون کتاب کراس راسه. آخرین باری که چک کردم کتاب تست هم براش نبود.

موفق باشی.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:** [محمد جواد تاتاری - ویرگول](https://virgool.io/@mjavadtatari)
**متن کامنت:**
ممنونم بابت راهنمایی هات.  
اگه میشه این 2 تا سوال منم جواب بده:  
1- نظرت رو در مورد ریاضی گفتی ولی حالا که ضریبش تغییر کرده، چیکار کنیم؟  
2- در مورد دروس اضافه شده، دیجیتال الکترونیک و سیگنال و سیستم، چه نظری داری؟ (بخصوص برای نرم افزاری ها)

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
سلام چه سوال سختی. من دفترچه ها رو نگاه کردم و یه چیزهایی به نظرم میرسه. فقط بگم من از فضای کنکوری کلا دور شدم و از نظر اساتید مطلع نیستم.

خب اول بگم که مشخصه که نمیشه همش رو زد. شما اینجوری فرض کن که ۹ تا ۱۰ تا درس رو بیشتر نمیشه زد. دیگه ته تهش ۱۱ تا (خیلی دارم ایده آل گرایانه میگم). درس‌های ضریب ۳ و ۴ که تکلیفشون مشخصه. از درس‌های ضریب ۲ اونایی رو باید برداری که بیشتر باهاشون راحتی. به نظرم بهتره با بچه‌ها هم صحبت کنی ببینید کدوم ها اصولاً سوالاتش راحت تره. یه نیم نگاهی هم داشته باشید که بعداً کدوم از درس‌ها به درد میخوره. میشه حتی ترکیبی برداشت چون نهایتا میانگین درصد رو میگیرن و فرقی نمیکنه. من به یکی از دوستان نرم افزاری ام میگم نظرش رو در مورد دروس نرم افزار اینجا بگه چون من واقعاً نمیدونم.

فقط یه چیز دیگه رو هم در نظر بگیرید که اهمیت هر درس به تعداد درس‌های هر گروهش هم بستگی داره. مثلاً نظریه و سیگنال تو یه گروه هستند و با زدن یکیشون میشه ۵۰ درصد نمره آورد در حالی که تو بقیه گروه‌ها که ۳ تا درس هستند هر کدوم احتمالاً ۳۳ درصد نمره بهت اضافه کنند. خدا رو شکر اینجور که من دیدم سال ۱۴۰۱ نسبت به ۱۴۰۰ فرقی نکرده پس میشه حدس زد که هر درس چند تا سؤال داره و چند درصد نمره رو شامل میشه.

در مورد هوش مصنوعی بگم که گرایش خودمه. مدار منطقی و معماری خیلی ربطی نداره (البته جدا مدارمنطقی درس ساده‌ای هست و میشه تست هاش رو زد). الکترونیک دیجیتال که کلاً برقیه. منم چون نخوندم و اصلاً آشنایی ندارم انتخابش نمیکنم (فرض میکنم شما مهندسی کامپیوتر بودی) ولی عوضش آمار و احتمال رو حتماً توصیه میکنم. تو هوش پایه ریاضی خیلی مهمه. همون اولی که وارد دانشکده میشید استاد راهنما با پرسیدن چند تا سؤال آمار و احتمال و سیگنال و خصوصا جبر خطی (که اینجا نداریم) میفهمه که شما چقدر پایه ات قویه و یکی از معیارهایی که انتخابت میکنه (یا نمیکنه) بر همین اساسه. مثال بزنم یه استادی از من سؤالات آمار و احتمال پرسید و چون من مباحثش رو به خاطر نداشتم قبولم نکرد. دروسی که تو هوش وجود داره تا دلتون بخواد داخلش انتگرال و امیدریاضی و اینجور مباحث هست پس خیلی راحت ترید اگر از همون اول با این مباحث راحت باشید.

در مورد شما هم صرفاً بگم که اگر به یادگیری ماشین علاقه دارید، آمار و احتمال رو حداقل بخونید. خیلی کاربردی و مهمه (آسون هم هست).

اگر بعداً چیز بیشتری به نظرم اومد مینویسم (فکر نمیکردم نوشتن یک جواب ساده، ۲ ساعت و اندی طول بکشه). امیدوارم جوابم به دردتون بخوره. هر چند با مرور مجدد فهمیدم چیز خاصی نگفتم.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:** [ShadiMardany - ویرگول](https://virgool.io/@m_42674117)
**متن کامنت:**
مرسی بابت راهنمایی یه عالیتون.من لیسانسم تمام شده و میخوام المپیاد شرکت کنم.راهی هست ک شرکت کنم؟

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
سلام. خسته نباشید.
تا اونجا که من میدونم راهی نیست.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:** [amoo behrooz - ویرگول](https://virgool.io/@amoobehrooz2030405060)
**متن کامنت:**
عالی بود دمت گرم کاش یه راه ارتباطی هم میدادی باهم بیشتر معاشرت میکردیم

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:** [فاطمه - ویرگول](https://virgool.io/@fatemeroshani7899)
**متن کامنت:**
سلام خیلی ممنون بابت مطلب خوبتون. فقط اینکه شما از فیلمای آموزشی دکتر یوسفی استفاده کردین یا فیلم های نکته و تست؟

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
سلام. خسته نباشید.

راستش هر دو!

**Comment by pourmand1376** [comment]
**نویسنده کامنت:** [حسامم - ویرگول](https://virgool.io/@hesamam)
**متن کامنت:**
فیلم های گام به گام با هوش مصنوعی استاد رهنمود رو من برای دسترسی بهتر به شکل پلی لیست در اوردم  
از این لینک میتونید استفاده کنید :  
[https://youtube.com/playlist?list=PLi-5CqLrw8iCekowYJp0dT0uWSVgWW71J](https://youtube.com/playlist?list=PLi-5CqLrw8iCekowYJp0dT0uWSVgWW71J)

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:** [محمد مرادی - ویرگول](https://virgool.io/@lpvhf110)
**متن کامنت:**
سلام. خیلی ممنون بابت به اشتراک گذاری.  
2 تا سوال داشتم از شما.  
1- با توجه به نکاتی که در مورد مدار منطقی و معماری گفتید، اگر برمیگشتید عقب و فیلم هایی که نام بردید رو نگاه کرده بودید، این درس رو توی المپیاد میزدید یا نه؟  
2- شما آشنایی با فیلم های دکتر شیرافکن دارید؟ اگر بله، توصیه اشون میکنید یا نه؟

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
سلام. متشکرم.
۱. بله من حتما آزمون معماری و مدارمنطقی رو شرکت می‌کردم.
۲. بله با ایشون آشنایی دارم. فیلم‌های نظریه و الگوریتمشون رو ۲سال پیش دیدم. تا جایی که یادمه ۴ تا کتاب رایگان هم منتشر کرده بودن که یکی‌اش سیستم عامل بود و برای اون زمان خوب بود‌. هر چند بعدها با کتاب‌های دقیق‌تر و بهتری آشنا شدم. من در جایگاهی نیستم که کسی رو توصیه کنم اما به نظر خودم فیلم‌هاشون عمق کافی رو نداره. توقع خیلی زیادی نداشته باشین. به هر حال هزینه فیلم هاشون نسبت به بقیه خیلی کمتره و برای اون هزینه مناسبه.

**Reply by pourmand1376** [reply]
**نویسنده کامنت:** [محمد مرادی - ویرگول](https://virgool.io/@lpvhf110)
**متن کامنت:**
سلام مجدد. ممنون بابت پاسخ گویی و اطلاعات ارزشمندتون.  
ببخشید یه چند تا سوال دیگه هم خدمتتون داشتم. من الان سال سوم هستم و خیلی جدی میخوام برای المپیاد اقدام کنم. میخواستم ازتون بپرسم برای المپیاد اگر برگردید (صرفا براي المپیاد نه کنکور) با دانش الان‌تون:  
1- چه کتاب هایی رو برای 4 تا درس 1-مدار منطقی 2-معماری 3-نظریه 4-سیستم عامل مطالعه میکردید؟ (شبکه رو نگفتم چون دانشگاه ما هر 2 ترم یه بار ارائه میشه و به المپیاد امسال نمیرسه که بخونم‌اش و به ناچار باید حذف كنم.)  
2- برای ساختمان و طراحی الگوریتم هم که خودتون گفتید فیلم‌های دکتر شریفی و دکتر گنج تابش رو ببینیم و بعد با کتاب دکتر قدسی و کورمن بریم جلو، درسته؟ (البته نفهمیدم که فیلم های آقای یوسفی هم برای المپیاد لازمه یا نه!؟)  
3- برای معماری هم برای المپیاد کل اون 4 سری فیلمی که گفتید لازمه؟

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
با سلام و احترام
عذرخواهی ‌می‌کنم که دیر جواب میدم.
۱- اگر قرار باشه برای المپیاد بخونید باید تمرین‌ها کتاب‌های مرجع رو خوب مسلط باشید معمولا. خوندن صرف کتاب هم به نظرم خیلی موثر نیست. تمرین حل کنید.
مدار منطقی اون زمان که من می‌خواستم بخونم بچه‌های امیرکبیر می‌گفتند کتاب حل‌المسائل موریس مانو کتاب خوبیه. معماری کامپیوتر: تجربه شخصی ندارم ولی با توجه به چیزی که بچه‌ها گفتند سوالات در حد تست‌های کنکور هست. البته یخورده طراحی هم داره که فکر می‌کنم اگر حل‌المسائل کتاب رو بخونید حتما موفق میشید.
نظریه زبان و ماشین: قطعا کتاب پیتر لینز + مسئله‌هاش. تو پرانتز بگم سوالات کنکور هم از همین مسائل هست اکثرا، پس می‌تونید سوالات کنکور رو حل کنید، فقط اثبات‌های لم تزریق (که اتفاقا سوال المپیاد بود) رو باید از همون کتاب بخونید. سیستم عامل: کتاب دکتر حقیقت. شبکه رو هم میگم برای اطلاع بقیه: فقط کراس راس.
۲- ببینید فیلم‌های آقای یوسفی رو نمیشه گفت لازمه، فقط میشه گفت کارتون رو راحت‌تر می‌کنه و زودتر یاد می‌گیرید. کلیتش همین هست که فرمودین در صورتی که وقت کافی دارین.
کورمن یخورده فهمش سخته. به نظرم اگر تست‌های کنکور یا سوالات کتاب دکتر قدسی رو حل کنید همون‌ مسائل رو لابه لاش یاد میگیرین. (وقتی سمت کتاب رفرنس برین که میخواین یه مطلب خاصی رو که نفهمیدین یاد بگیرین، نه این که لزوما کل درس رو رفرنس بخونید. انقدرها وقت نداریم.)
۳- نه لازم نیست. من فقط اینجوری پیش رفتم. کاملا هم رندوم فیلم‌ها رو پیدا می‌کردم و با اساتید آشنا می‌شدم و باهاشون جلو می‌رفتم. اصلا و ابدا فکر نکنید این‌ها رو برنامه ریزی پیش بردم که اول این رو ببینم بعد اون یکی رو. زندگی رو کلا خیلی نمیشه برنامه ریزی کرد. :) خودش پیش میره.
به نظرم برای کنکور که فیلم‌های آقای یوسفی کفایت میکنه. برای المپیاد احتمالا دیدی که دکتر گودرزی نسبت به معماری بهتون میده خیلی بهتره.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:** [محمدیان - ویرگول](https://virgool.io/@m_30369605)
**متن کامنت:**
خیلی ممنون بابت اشتراک گذاری تجربیات مفیدتون.میشه بیشتر برای المپیاد راهنمایی کنید برای کتاب خیلی سرچ کردم همش برای المپیاد کامپیوتره نه مهندسی کامپیوتر.

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
سلام. تو مطلبی جداگانه در مورد المپیاد مهندسی کامپیوتر نوشتم. لطفا اون رو بخونید

**Comment by pourmand1376** [comment]
**نویسنده کامنت:** [Alizolfi - ویرگول](https://virgool.io/@Alizolfi14)
**متن کامنت:**
سلام مهندس  
کل اطلاعات من از کامپیوتر اینه که میتونم روشنش کنم  
اما  
این متن جدای از محتوای تخصصی‌اش(که بنده چیزی ازش متوجه نشدم?)،پر از انگیزه و امید بود،مخصوصا توی این روزا که فقط خبر بد میشنویم.  
خیلی مبارک باشه،چقدر خوشحال شدم از۱.رتبه یک شدن شما۲.خوندن این متن پر از شور و اشتیاق،تنت سالم و دلت خوش.  
همیشه بدرخشی.

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:**[محمدهادی - ویرگول](https://virgool.io/@hadi.mohamaditabar) 
**متن کامنت:**
خوب بود. ممنون  
فیلم های دکتر شیرافکن هم حرف نداره

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:[Mohamad\_Akbari - ویرگول](https://virgool.io/@Motech)
**متن کامنت:**
سلام روزت بخیر  
خیلی خوب و قشنگ نوشتی تجربیات و خاطره هات رو، واقعا استفاده کردم و البته خودم هم تجربه تمام این مسائلی رو که گفتی داشتم و تقریبا 80 درصد شبیه به خودت عمل کردم اما خب شرایط زندگی خیلی توی مبحث درس و روز کنکور تاثیر گذاره، واقعا روز کنکور دوره کردن اون همه مطلب در زمان کم خیلی کار هنرمندانه و دشواریه چون تجربه اش رو داشتم میگم اینو، در کل امیدوارم بعد از اینهمه فراز و نشیب و انتخاب دانشگاه مورد نظرت، اون دروس و اساتید مناسبی که از قبل توی ذهنت بوده رو تجربه کنی و رضایت خوبی از دانشگاهت داشته باشی، من الان دانشجوی کارشناسی نرم افزار هستم و اصلا از سطح دروس دانشگاهی و اساتید راضی نیستم و دانشگاه برام یه محیط وقتگیر شده،زیاد گلگی نمیکنم اما امیداورم همیشه موفق و شاد باشی.

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:[یادداشت های یک دختر معمولی 2 - ویرگول](https://virgool.io/@yaddashtha)
**متن کامنت:**
سلام خیلی ممنون عالی بود، از کی شروع به خوندن کردین؟

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
سلام
تقریبا از اوایل آبان با ساعت مطالعه خیلی کم (۳ ۴ ساعت) شروع کردم و از آذر دیگه به طور جدی مطالعه می‌کردم (نه خیلی زیاد. ۵ ۶ ساعت). بعد کم کم ساعت مطالعه‌ام رو بردم بالا.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:[hanieradaie - ویرگول](https://virgool.io/@hanieradaie)
**متن کامنت:**
سلام،  
مرسی بابت توضیحات خوبی که نوشتین. تجربه هایی که در رابطه به المپیاد ذکر کرده بودین خیلی به درد بخور بودن. یه سوالی در همین رابطه داشتم، شما میدونید که کسی که بخواد از طریق آزمون ارشد برای المپیاد اقدام کنه و سال چهارمی ام باشه دقیقا چه مراحلی رو باید طی کنه ؟(علی الخصوص که الان به خاطر کرونا تنها راه شرکت تو المپیاد همون آزمون ارشد هست)

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
سلام.
کار خاصی نباید انجام بدین. همین که رشته‌تون مهندسی کامپیوتر باشه و رتبه خوب در ارشد بیارین خود سنجش پیام میده و دعوتتون می‌کنه. البته نمی‌دونم بچه‌های علوم کامپیوتر میتونن شرکت کنن یا نه.

یه تیک هم در زمان ثبت نام کنکور ارشد داریم که میگه منحصرا برای المپیاد می‌خوام شرکت کنم که اون رو شما نباید بزنین. سال سومی‌ها که فقط برای المپیاد می‌خوان در ارشد شرکت کنند اون تیک رو می‌زنن.

**Reply by pourmand1376** [reply]
**نویسنده کامنت:[hanieradaie - ویرگول](https://virgool.io/@hanieradaie)
**متن کامنت:**
سنجش در بخش توضیحات تکمیلی اش برای ثبت نام نوشته که سال چهارمی ها " بند مربوط به منحصرا المپیاد، در صفحه ثبت نام نیز مخصوص دانشجویان سال سوم مقطع کارشناسی می باشد و دانشجویان سال چهارم نیاز به پر کردن بند منحصراً المپیاد ندارند و به دانشگاه مراکز مناطق مراجعه نمایند . "
شما میدونید که منظور از مراجعه به دانشگاه مراکز یعنی چی ؟ و اصلا برای ثبت نام لازم ه ؟

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
والله نمیدونم منظورش چی بوده. چیزی که می‌دونم اینه که تا قبل کرونا افراد به ۲ صورت می‌تونستن به المپیاد وارد بشن: ۱۵ نفر اول کنکور و ۵ نفر از هر منطقه (۱۰ تا منطقه داریم که میشه ۵۰ نفر).
کنکور که مشخصه. برای منطقه اول باید در سطح دانشگاه انتخاب بشین که معمولا مسئول استعداد درخشان یه آزمونی برگزار میکنه و ۵ نفر رو برای منطقه انتخاب میکنه. بعد در هر منطقه آزمون برگزار میشه و ۵ نفر اول هر منطقه میرن به مسابقه کشوری. برای دومی نیاز هست با مسئول مربوطه صحبت کنید و خودش کارهای ثبت نام در المپیاد رو براتون انجام میده.
البته امسال المپیاد منطقه‌ای برگزار نشد و افراد صرفا از کنکور انتخاب شدند و بعدش هم کلا المپیاد کنسل شد. نمی‌دونم رویکرد سال بعد به چه صورتی خواهد بود.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:[پرهام بیچرانلو - ویرگول](https://virgool.io/@Parham_bh)
**متن کامنت:**
سلام خیلی مطلب خوبی بود ممنون.  
یک سوالی که دارم اینکه شما چه جوری درس دانشگاه و کنکور رو هم‌زمان پیش می‌بردید؟ چون درس‌هایی که باید امسال پاس کنم هیچ کدوم تو کنکور نمیاد و از اون‌طرف خب برام معدلم مهمه. شما چه پیشنهادی می‌دید؟  
در ضمن تبریک میگم بابت رتبتون.

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
یه Trade-off هستش این مطلب. باید ببینید کدومش بیشتر براتون ارزش داره و چقدر بیشتر ارزش داره.

من خودم سعی می‌کردم تا حد ممکن فعالیت‌های دانشگاه رو مینیمم کنم. زیاد روی تکالیف و تمارین وقت نمی‌گذاشتم و درس‌ها رو هم میذاشتم شب امتحانی بخونم (کاری که تقریبا تو کل دوران لیسانس انجام دادم. من کار میکردم و اصلا فرصت نمیشد درس بخونم).

کلاس‌ها هم مجازیه و عملا نسبت به حضوری وقت زیادی براتون سیو میشه.

البته نمی‌دونم معدلتون در چه سطحی است. اگر معدل خیلی بالا (۱۸ به بالا) دارین و از نظر رتبه معدل تو یکی دونفر اول رشته‌تون هستید، می‌تونید بدون کنکور از طریق استعداد درخشان هم وارد دانشگاه‌های برتر بشین. در اون صورت احتمالا ارزش حفظ معدل بیشتر کنکور میشه.

**Comment by pourmand1376** [comment]
**نویسنده کامنت:[حسین - ویرگول](https://virgool.io/@zeynali)
**متن کامنت:**
مرسی از اینکه تجربیاتت رو به اشتراک گذاشتی. نکته های خوبی داشت.  
راستی برای دیتا بیس وقت کم آوردی ؟ برام درصدی که زدی عجیب بود چون درسای سخت رو اینقدر خوب زدی بعد درس آسون دیتا بیس از همه کمتر شده!  
کلا راجع به روش جمع بندی درسا و مدیریت خود جلسه هم بگی مفید هست بخصوص اینکه تو ویدیوی آپارات گفتی 2 بامداد تازه سی پی یوت اورکلاک میشده :دی و احتمالا شبا درس می خوندی روز کنکور مشکلی برات پیش نیومد؟ خوابالوده نبودی؟  
و اینکه هر فصل رو طبق بودجه بندی وقت مطالعه گذاشتی یا جور دیگه ؟ ممنون میشم کمی بیشتر توضیح بدی.  
امیدوارم درادامه زندگیت هم موفق باشی .  
برات آرزوی خوشبختی دارم

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.

**Reply by pourmand1376** [reply]
به به. چقدر نکته سنج و دقیق هستید.
شب کنکور ۳ ۴ ساعت بیشتر نتونستم بخوابم. از ۵ صبح بیدار شدم و به در و دیوار نگاه می‌کردم. دیگه خوابم نبرد. امتحان هم ۸:۳۰ تا ۱۳:۰۰ بود. تقریبا ساعت‌ ۱۲ بود که چشمام درد گرفت و خیلی زیاد خوابم میومد. دیتابیس و زبان رو خیلی غلط غلوط زدم. دیتابیس، یخورده بی دقتی بود، یخورده هم غرور. سوالات دیتابیس اتفاقا ساده هم بود، من کوئری‌ها رو که می‌دیدم یاد کوئری‌های پیشرفته‌تری می‌افتادم که قبلا تو شرکت می‌نوشتیم. خسته هم شده بودم دیگه، درست حسابی سوالا رو نخوندم و این شد که شد. بعد جلسه که رسیدم خونه، زنگ دوستم سینا زدم و هم نظر بودیم که کنکور سختی بود، مثل ماراتن بود. جواب دادن این حجم سوال تخصصی در اون بازه زمانی کوتاه کار طاقت‌فرسایی هست.

من تکنیک زمان‌بندی خاصی استفاده نکردم. همون روش معمولی همیشگی. سوالاتی رو که بلد نیستم ضربدر می‌ذارم و اونایی که بعدا می‌خوام حل کنم خط تیره میذارم. اینطور که فهمیدم بهتره سوالاتی که شک داریم رو بعدا در موردش فکر کنیم. انگار یه Thread از مغز جدا میشه و سعی می‌کنه تو اعماق ذهن دنبال جواب بگرده. جالبه مثلا سوالات نظریه رو که حل می‌کردم یهو جواب سوال قبلی رو یادم میومد و سریع میرفتم میزدم و برمیگشتم.

بعد از حل دیتابیس و زبان، یه سری سوالاتی که شک داشتم یا حل نکرده بودم رو نگاه کردم. فکر کنم ۱۰ دقیقه بیشتر وقت نشد. وقت به شدت تنگ بود. در نهایت امر اون چیزی که باعث میشه سرجلسه کنکور راحت سوالات رو بزنین و وقت کم نیارین، تسلط رو مطالب درسیه، نه فلان تکنیک مدیریت زمان. مثلا وقتی می‌خواستم الگوریتم بخونم غمم نبود رو یک سوال حتی نیم ساعت فکر کنم یا سوالات رو با روش خودم حل کنم. خیلی مهمه که سوالات رو خودتون حل کنید و وقت براش بذارین.

امسال چون تجربه کنکور ۹۹ وجود داره، این آزمون جامع ها برای این که به زمان‌بندی کنکور عادت کنید، بد نیست. من میخواستم یکی دوتاش رو شرکت کنم ولی دیدم خیلی گرونه و در ثانی خود موسسات هم هیچ‌کدوم نمی‌دونستن تعداد سوالات و زمان امتحان قرار بود چجوری باشه، این بود که شرکت نکردم. امسال شرایط از این نظر بهتره و می‌تونین بهتر آماده بشین ولی بازهم به نظرم آزمون مثل قلم‌چی که هر دو هفته یه بار امتحان بدیم خیلی جالب نیست.

درباره جمع بندی هم بهتون بگم که هفته‌های آخر آزمون کتاب‌های تست رو باز می‌کردم و از تمام فصل‌ها چند تا تست حل می‌کردم و انگار تو یه پارت کل درس مرور میشد. با همون سوال حل کردن می‌فهمیدم کجای کار میلنگه و همون مبحث رو دوباره یه نگاه می‌انداختم. نکته‌های استاد یوسفی (که شب قبل کنکور گذاشتن) هم خوب بودن ولی کاش زودتر می‌خوندم نه شب کنکور.
خودم یه سری درسا خلاصه نوشتم ولی خیلی مفید نبود به نظرم. میدونید خلاصه برای من یه سری کلمه کلیدی بود که قاعدتا همش بدیهیه. اصلا خلاصه مطالب هر کتابی رو دربیارین یه سری کلمات بدیهیه که همه میدونن. تو جزئیات هست که آدم گیر میکنه نه کلیات. آقای شعبانعلی در این مورد بهتر توضیح دادند. همین شد که خلاصه‌هایی که نوشته بودم (با این که به نظرم دقیق بود) خیلی به دردم نخورد.

درباره بودجه بندی هم راستش من به اون صورت توجه نکردم. فقط درسایی که می‌خواستم یه سری فصل‌هاش رو کمتر بخونم یا حذف کنم طبق تعداد سوالات سال‌های گذشته استناد می‌کردم. مثلا فصل ۵ شبکه یا فصل Planning هوش رو می‌خواستم کمتر بخونم طبق همین کمتر می‌خوندم. الان هر چی فکر می‌کنم چیز بیشتری به ذهنم نمیاد که بهتون بگم.

**Reply by pourmand1376** [reply]
**نویسنده کامنت:[حسین - ویرگول](https://virgool.io/@zeynali)
**متن کامنت:**
ممنون بابت جواب دقیق و سریعی که دادی .  
واقعا جالب بود که با وجود بی خوابی اینقدر خوب از پس ماراتون سوالها بر اومدی. ردبولی ، هایپی چیزی لازم داشت:دی  
در ضمن خیلی خوبه که تو این سن علاوه بر پایه نظری خوب، تجربه برنامه نویسی تو شرکت هم داری ، به امید خدا آینده خوبی تو این رشته خواهی داشت.  
برات آرزوی موفقیت در همه ی ابعاد زندگیت دارم. شاد و سلامت باشی

----
این کامنت از سایت ویرگول که قبلا استفاده می‌کردم استخراج شده است.
